zebrad/src/commands/start.rs

```diff
@@ -90,6 +90,9 @@ impl StartCmd {
             ChainSync::new(&config, peer_set.clone(), state.clone(), chain_verifier);
 
         info!("initializing mempool");
+
+        let (transaction_sender, transaction_receiver) = tokio::sync::watch::channel(None);
```

Comment:
Readability: can you rename these channels so it's clear they're for the mempool?

Reply:
I am not sure if you mean change the name of this variables just here or everywhere. Also not sure if you want `mempool_transaction_sender` as the name or what exactly. In my opinion this names are pretty clear as they can be found always in context but open to change them if i know specifically to what.